modules/search/class.jetpack-search-helpers.php

```diff
@@ -227,6 +227,10 @@ static function get_date_filter_type_name( $type, $is_updated = false ) {
 	static function generate_widget_filter_name( $widget_filter ) {
 		$name = '';
 
+		if ( ! is_array( $widget_filter ) || ! array_key_exists( 'type', $widget_filter ) ) {
+			return $name;
+		}
```

Comment:
Are we bailing out and returning `''` if `$widget_filter` is malformed?

Reply:
Yep - that's the existing behaviour. It currently just drops through the `switch` if there's no `$widget_filter['type']`.